client/src/pages/login/adminDash/admin.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './admin.css';
+
+function Admin() {
```

Comment:
Change to a class, because we will be working with states later on
```suggestion
class Welcome {
  render() {
    return <h1>Hello, {this.props.name}</h1>;
  }
}
```

Reply:
Ignore this

Reply:
I changed them to arrowed functions. Would you still like me to change them to classes?

Reply:
Yes, because we will be working with the React state for this component

Reply:
Just changed them to state classes that extend `React.Component`